test/integration/screenlock.nix

```diff
@@ -81,5 +102,15 @@ in {
         "  • Checks completed.\n"
     )
     assert out == expected, f"Expected did not match actual, got \n{out}"
+
+
+    # Test sway, swaylock is disabled by default
+    status, out = sway.execute("paretosecurity check --only 37dee029-605b-4aab-96b9-5438e5aa44d8")
+    expected = (
+        "  • Starting checks...\n"
+        "  • Access Security: Password is required to unlock the screen > [FAIL] Password after sleep or screensaver is off\n"
+        "  • Checks completed.\n"
+    )
+    assert out == expected, f"Expected did not match actual, got \n{out}"
```

Comment:
Missing test for enabled swaylock and passing check.

Reply:
yep :), Waiting for few examples as there are multiple ways how to configure it.